ui/pages/swaps/slippage-buttons/slippage-buttons.js

```diff
@@ -78,9 +83,14 @@ export default function SlippageButtons({
           'slippage-buttons__header--open': open,
         })}
       >
-        <div className="slippage-buttons__header-text">
+        <Text
+          data-testid="slippage-buttons-header-text"
+          variant={TextVariant.bodySmBold}
+          marginRight={2}
+          color={TextColor.primaryDefault}
+        >
```

Comment:
This results in a `p` tag which shouldn't be used inside a `button` 

This looks like an opportunity to utilize the [ButtonBase with end Icon component](https://metamask.github.io/metamask-storybook/?path=/story/components-componentlibrary-buttonbase--end-icon-name) and to add the additional styling

Reply:
Not sure if Devin will be able to handle this change. I think we can keep the components to `Text` and change the rendering element to a span as you suggested below